openedx/features/pakx/lms/pakx_admin_app/views.py

```diff
@@ -7,9 +7,12 @@
 from rest_framework.response import Response
 
 from .constants import GROUP_TRAINING_MANAGERS, ADMIN, STAFF, TRAINING_MANAGER, LEARNER
+from openedx.core.djangoapps.user_authn.views.registration_form import AccountCreationForm
```

Comment:
why are we using this? please check the Figma link for design for user registration, follow that design for requirements.

Reply:
??